# Exception Handling
# Abnormal termination

print(10/0)
try:
    print('Micky-1')
    print('Micky-2') 
    print(10/0)        
except:
    print('Exception raised due to : ')    

print()
print('Rest of the code...')


